README.md

```diff
@@ -40,8 +40,8 @@ Range, Readings
 
 Establish quality parameters: 
 
-- What is the maximum complexity (CCN) per function? _enter CCN and create corresponding yml in the `.github/workflows` folder
-- How many lines of duplicate code will you tolerate? _enter the number of lines and create corresponding yml in the `.github/workflows` folder
+- What is the maximum complexity (CCN) per function? 3 CCN and create corresponding yml in the `.github/workflows` folder
+- How many lines of duplicate code will you tolerate? 50 the number of lines and create corresponding yml in the `.github/workflows` folder
```

Comment:
You don't tolerate duplicates of more than 3 lines in the yml file. It's not 50

Reply:
Oops... Mistakenly, I have entered max. no. of lines per file. Yes, my goal was to reduce duplicates not more than 3 lines. I have achieved as of now.

Reply:
Will correct it in .readme file.